packages/client/src/index.ts

```diff
@@ -277,7 +288,7 @@ export default class LogtoClient {
     this.accessTokenMap.set(accessTokenKey, {
       token: accessToken,
       scope,
-      expiresAt: Math.round(Date.now() / 1000) + expiresIn,
+      expiresAt: requestedAt + expiresIn,
```

Comment:
better to add some comments to the type definition of this field to clarify it's an estimate?

Reply:
OK